test/runtests.py

```diff
@@ -99,9 +103,19 @@ def test_path(folder, is_file):
                 baseline = os.path.splitext(js_file)[0] + '.baseline'
                 baseline = os.path.join(folder, baseline)
                 if os.path.isfile(baseline):
-                    expected_output = None
+                    baseline_output = None
                     with open(baseline, 'r') as bs_file:
-                        expected_output = bs_file.read().replace('\r', '')
+                        baseline_output = bs_file.read()
+
+                    # clear carriage return
+                    # todo: remove carriage return at the end of the line
+                    #       or better fix ch to output same on all platforms
+                    expected_output = baseline_output.replace('\r', '')
+
+                    # normalize tabs on both outputs to 2 spaces
+                    expected_output = expected_output.replace('\t', '  ')
```

Comment:
> 2020 2061 7420  Error: xx.   at .. there is no 09 here..
> 
> update:
> This is an output on Linux Ubuntu for throw new Error("xx")

Is it possible due to missing command line flag `-ExtendedErrorStackForTestHost`? If you add that flag, output should have `\t`. That flag is default in test harness and also now in runtests.py. (I don't know the history why this flag was added and affects this behavior.)

```
if (CONFIG_FLAG(ExtendedErrorStackForTestHost))
{
    bs->AppendChars(_u("\n\tat "));
}
else
{
    bs->AppendChars(_u("\n   at "));
}
```